src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -411,6 +437,13 @@ public CodeFixProvider GetSuppressionFixer(string language, ImmutableArray<Diagn
             return false;
         }
 
+        private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict CodeFixes in Interactive
+            return provider is AddImport.AbstractAddImportCodeFixProvider ||
```

Comment:
Well, we're not succeeding at our goal of a consistent experience by providing some. I suspect the goal might need to be questioned here...